lua/lir/actions.lua

```diff
@@ -153,14 +154,34 @@ end
 function actions.rename()
   local ctx = get_context()
   local old = string.gsub(ctx:current_value(), sep .. "$", "")
-  local new = vim.fn.input("Rename: ", old)
+  local default = ""
+  if config.values.rename_default then
```

Comment:
The reason I added that is so the user can decide if they want the input filled with the old filename or cleared.

I prefer to have it empty and retype the new name rather than <C-w> a few times to get rid of what I don't want

Reply:
Change it so that it is received as a function argument, instead of being held in config.

This is because we want to implement the options related to action to be received by the function.

```lua
function actions.rename(rename_default)
  ...
```